src/styles/blocks/header.scss

```diff
@@ -0,0 +1,83 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  
+  font-weight: 500;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+
+  &__logo:hover {
```

Comment:
```suggestion
  &:hover {
```
write it inside &__logo
<img width="1044" alt="image" src="https://user-images.githubusercontent.com/102413088/177989234-f1471703-0db7-4b93-ab25-c3b3ec5f3d43.png">